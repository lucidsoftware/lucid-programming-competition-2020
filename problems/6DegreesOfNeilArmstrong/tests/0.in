Bismuth Rotaxian
8
Bismuth Rotaxian -> Garfunkel Torol Booth
Rhaj Dasgupta -> Garfunkel Torol Booth
Garfunkel Torol Booth -> Bob Behnken
Peggy Whitson -> Neil Armstrong
Garfunkel Torol Booth -> Peggy Whitson
Buzz Aldrin -> Kevin Bacon
Bob Behnken -> Buzz Aldrin
Buzz Aldrin -> Neil Armstrong