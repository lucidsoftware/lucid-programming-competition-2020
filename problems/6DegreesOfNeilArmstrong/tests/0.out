Neil Armstrong
Peggy Whitson
Garfunkel Torol Booth
Bismuth Rotaxian